lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1215,6 +1215,9 @@ void LocalEnforcer::complete_termination(
   reporter_->report_terminate_session(termination_req, logging_cb);
   events_reporter_->session_terminated(imsi, session);
 
+  // clear all metrics associated with this session
+  session->clear_session_metrics();
```

Comment:
should we do the same on session creation to cover the case when session is not terminated properly?

So when we do 
```
auto session_map     = session_store_.read_sessions({imsi});
```
If session_map is lenght 0, then trigger that function to make sure there is no left over for that imsi.

Reply:
What do you mean by `when session is not terminated properly` ?

Reply:
Imagine we exit in line 1211. WE will never clear that metric. So it would be there forever (i guess reboot would clear it).

So maybe it would be good to make sure that when we start a session fresh, we do not have anything for that session in the metrics. It could be a simple way to reconcile both states for sure.

Reply:
oooo I see. But if fail at line 1211, we don't delete the SessionState object. So there could be another opportunity for the state to be removed later on. 
I was thinking about cleaning up on Session creation as well. But since the metric specifies sessionID as one of its label, I can't easily remove it without changing our metric hashing implementation.  
So right now, the lifecycle of the metrics is tied to the SessionState object. If we have a trouble cleaning up the session in SessionD, the metric will never get cleared :/